src/modules/RF69/RF69.cpp

```diff
@@ -762,6 +787,13 @@ int16_t RF69::setEncoding(uint8_t encoding) {
   }
 }
 
+int16_t RF69::setLnaTestBoost(bool value) {
```

Comment:
New public methods should have a keyword definition in keywords.txt (make sure to use true tabs and not spaces in that file).